.github/workflows/daily-build.yml

```diff
@@ -18,25 +21,38 @@ jobs:
           distribution: 'adopt'
           java-version: 11
 
+      - name: Set environment variable
+        if: github.event.action == 'check_connector_for_breaking_changes'
+        run: |
+          echo "BUILD_USING_DOCKER=true" >> $GITHUB_ENV
```

Comment:
Lets change this flag to gradle property instead of env variable

Reply:
Updated